### ISSUE TITLE
Env/ Config File Not Found

### ISSUE THREAD

**Issue by Ayush7445** [joshcoales/telegram-auto-forwarder#1]
This code doesn't have config/env file add it to make it usable

**Comment by SpangleLabs** [comment]
Heh, yeah, I threw this together a bit.
It requires a config file, config.json like so:
```
{
  "api_id": 00,
  "api_hash": "---"
}
```


And a forwarders file, forwarders.json:
```
{
  "forwards": [
    {
      "source": "123456",
      "destination": "234567"
    }
  ]
}
```

And then it will update the `forwarders.json` file with latest_id values as it runs.

I should put this in a readme, but it was just a quick little script I threw together

**Comment by Ayush7445** [comment]
Yeah you should add all file and make it Fully deployable on Heroku.

So someone can deploy it and use this.

**Comment by SpangleLabs** [comment]
I've never used Heroku to be honest. I just run this script on my home server
